sdk/nodejs/autoscaling/metrics.ts

```diff
@@ -21,31 +21,24 @@
 //
 // These give a better developer experience and are just sugared strings.
 
+import {MetricsGranularity, Metric} from ".";
 
-import * as cloudwatch from "../cloudwatch";
-
+// @deprecated Use MetricsGranularity.OneMinute instead.
```

Comment:
These need to use JSDoc comments (with the `/` and two `**`, e.g. `/** @deprecated Use MetricsGranularity.OneMinute instead. */`)

Reply:
Thanks for catching these